kbchat/kbchat.go

```diff
@@ -641,3 +650,21 @@ func (a *API) LogSend(feedback string) error {
 
 	return a.runOpts.Command(args...).Run()
 }
+
+func (a *API) Shutdown() error {
+	if a.runOpts.Oneshot != nil {
+		err := a.runOpts.Command("logout", "--force").Run()
```

Comment:
--force? (will break nopw accounts if this is the only login)

Reply:
That's [how the TS bot does it](https://github.com/keybase/keybase-bot/blob/master/src/service/index.ts#L98). To be fair though, that was likely written before no pw accounts. I'm happy to change.